src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs

```diff
@@ -1,20 +1,31 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+
 namespace Microsoft.CodeAnalysis.Classification
 {
     public static class ClassificationTypeNames
     {
+        /// <summary>
+        /// Additive classifications types supply additional context to other classifications.
+        /// </summary>
+        public static ImmutableList<string> AdditiveTypeNames { get; } = ImmutableList.Create(StaticSymbol);
```

Comment:
Why is this an `ImmutableList` over an `ImmutableArray`?

Reply:
There is no good reason. Will change.